### ISSUE TITLE
User with specified identity already exists

### ISSUE THREAD

**Issue by sterlp** [Azure/api-management-developer-portal#2142]
**Every bug report should have precise description and reproduction steps; console traces or source code references are appreciated.**

**For assistance requests, contact Azure support or submit a post [on Stack Overflow](http://aka.ms/apimso). We don't provide support through GitHub Issues. Feature requests can be raised [on the Azure Feedback Forum](https://aka.ms/apimwish).**

## Bug description

Azure AD login to developer portal creates an "User with specified identity already exists". Login with AD app was successful.

Same issue as here: https://github.com/Azure/api-management-developer-portal/issues/480

## Reproduction steps

0 create ad app with id login
![image](https://user-images.githubusercontent.com/774984/228262839-8bacf367-48ab-4a94-8e0e-58e69a30835e.png)

1. Manually configure the ad integration: https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-aad?WT.mc_id=Portal-fx
2. Delete Username and Passwort Identity Provider
![image](https://user-images.githubusercontent.com/774984/228260416-7121760a-52de-4b35-8dab-f557d5fcce78.png)
3. Publish developer portal
4. Login into the developer portal with AD

![image](https://user-images.githubusercontent.com/774984/228260026-24a998a0-fc6a-4ea3-ba1d-f1589ac4e19a.png)

![image](https://user-images.githubusercontent.com/774984/228262188-32dbcb05-41fb-40d2-bb2b-e17175a6461f.png)

## Expected behavior

Login works user from ad is accepted.

## Is your portal managed or self-hosted?

Managed 

## Release tag or commit SHA (if using self-hosted version)

[e.g., release `2.0.0`, commit `c45da9778b70d369aba60fa2e63c191efe2b548f`]

## API Management service name

[e.g., contoso-api]

## Environment

 - Operating system: [e.g., iOS]
 - Browser: [e.g., Google Chrome, Safari]
 - Version: [e.g., 22]

## Additional context

Add any other context about the problem here, including screenshots.

**Comment by sterlp** [comment]
I have a fun fact - I tried the legacy portal - which works in this case a bit better:
![image](https://user-images.githubusercontent.com/774984/228533667-2a4ef91e-9ed3-47fe-ad2c-82596b4fea9a.png)

Means here the login works and I see the "registration" here I can see that I am already regisered with my email; what ever this means. I just changed the email a bit and it works, I can login.

Means the bugs seems to be, that I am the admin of the thing and in addition I just wanted to test with my account the AD login, which doesn't work in the "new portal" but is wotking with a workaround in the old.

Same bug; but no workaround in the new portal.

**Comment by javierbo1989** [comment]
Hi Paul, I have created a work item to investigate this, we will keep you posted. Thanks!

**Comment by rcatrina** [comment]
following this thread for the linked support request ,pending an update from APIM PG here

**Comment by sterlp** [comment]
Is where an update yet to this issue?

**Comment by javierbo1989** [comment]
Hi folks, thank you for contacting us regarding this issue. We apologize for any inconvenience caused by the delay and appreciate your patience while we investigate the issue. Our team is currently reviewing your case and we will get back to you with an update as soon as possible.

**Comment by ccrudolabs** [comment]
This began to happen in the middle of last week and this time, it is possible that they have applied some change, as is in the deprecated portal, this has been working for a long time, it does not matter if you are an administrator, they applied changes, it affects the proper functioning, I recommend a rollback. I estimate they will solve it soon

**Comment by sterlp** [comment]
This would be good, this issue is already 2mo old. Thanks man.

**Comment by sbenji77** [comment]
Hi @sterlp, I've just come across your post, the only thing that solves this error message is to delete the user from the APIM and reauthenticate, this recreates the user in the apim and removes this error message. 
We've never had a response either

**Comment by sterlp** [comment]
Hello @sbenji77,

thanks for your response. As mentioned your solution isn't possible. As I am not in the user list. I am the "admin" and the owner of the subscription. Means if you use the old UI you will see, that my account can't be created because the admin is also part for some reason of the users, but the account can't be created.

Was here a change? I can double check on monday, but as I create the ticked I already checked if my account is where.

What do you mean by "We've never had a response either"? I see no other response, despite yours... .

Cheers

Paul

**Comment by sterlp** [comment]
Hey guys,

not a real solution but if I delete myself and if I use a system account for the deployment of the APIM, then it starts working:
![image](https://github.com/Azure/api-management-developer-portal/assets/774984/176b5dbd-a883-4dda-afca-a563483c6aeb)

For me it is still broken // as this is basically the workaround I described here:
https://github.com/Azure/api-management-developer-portal/issues/2142#issuecomment-1488504327

In short I cannot use the developer portal if I create with my account APIM; seems still to be that why. You don't get any proper error in the new UI, only if you use the old one.

Cheers

Paul

**Comment by mrcarlosdev** [comment]
This issue is related to managed developer portal. We advise you to create a Azure support request to get assistance on this issue. Please refer to the below link to create a new Azure support request, Please select Problem Type = "Developer Portal" in the request to route it appropriately.

https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request
